pyibisami/ami_parameter.py

```diff
@@ -314,7 +314,7 @@ def __init__(self, name, tags):
                     temp_vals = list(map(float, vals))
                 except (ValueError, TypeError):
                     raise AMIParamError("Couldn't read floats from '{}'.\n".format(vals))
-            elif param_type in ("Integer", "Tap"):
```

Comment:
The removed ', "Tap"' should be added to line 312.

Reply:
[serdes_tx.ami.txt](https://github.com/capn-freako/PyAMI/files/6273727/serdes_tx.ami.txt)

This is the file I sent you a few days ago, Taps are specified with type `Range`.
The file is generated with Matlab's SerDes Toolbox. 
The [ibis spec[](https://ibis.org/ver7.0/ver7_0.pdf) (Section 10.11.1) also mentions the type Range, that's why I moved the Tap out from the List type.

Did I understand something wrong here?

Reply:
I'm having trouble understanding (remembering?) what was going on here.